internal/wasm/jit/engine.go

```diff
@@ -159,12 +159,12 @@ type (
 		// Set when making function call from this function frame.
 		returnStackBasePointer uint64
 		// Set when making function call to this function frame.
-		compiledFunction *compiledFunction
+		compiledFunction *compiledFunctionInstance
 		// _ is a necessary padding to make the size of callFrame struct a power of 2.
 		_ [8]byte
 	}
 
-	compiledFunction struct {
+	compiledFunctionInstance struct {
```

Comment:
same thing on the docs on compiledFunction vs compiledFunctionInstance as it is a little confusing